python/pyspark/sql/functions.py

```diff
@@ -112,11 +112,8 @@ def _():
     'sinh': 'Computes the hyperbolic sine of the given value.',
     'tan': 'Computes the tangent of the given value.',
     'tanh': 'Computes the hyperbolic tangent of the given value.',
-    'toDegrees': 'Converts an angle measured in radians to an approximately equivalent angle ' +
-                 'measured in degrees.',
-    'toRadians': 'Converts an angle measured in degrees to an approximately equivalent angle ' +
-                 'measured in radians.',
-
+    'toDegrees': '.. note:: Deprecated in 2.0, use degrees instead.',
```

Comment:
I will create a separate issue because SPARK-16588 is fixed in 2.0.0 if it makes sense. Sorry, I was just confused.